tests/components/hydrawise/test_switch.py

```diff
@@ -34,7 +34,7 @@ async def test_states(
 
     auto_watering2 = hass.states.get("switch.zone_two_automatic_watering")
     assert auto_watering2 is not None
-    assert auto_watering2.state == "on"
+    assert auto_watering2.state == "off"
```

Comment:
Why did this change?

Reply:
Because of https://github.com/dknowles2/pydrawise/pull/115/. I will undo and wait for the fix to be released.

Reply:
Reverted. But the tests are currently failing until we update to the new version of `pydrawise`.